README.md

```diff
@@ -0,0 +1,38 @@
+# FakedPeopleBot
+
+This program is a group study project for ITMO University. </br> It allows to generate fake people's names, surnames and addresses. </br>
+
+## Installing dependencies
+To install dependencies run this commands:</br>
+``` $ pip3 install discord.py ```</br>
+``` $ pip3 install Faker ```
+
+## Set up settings in [config](https://github.com/turingsquad/FakedPeopleBot/tree/master/config) files
+
+To do this you only need to change token in [config.ini](https://github.com/turingsquad/FakedPeopleBot/blob/master/config/config.ini) file.</br>You should paste a token of your bot there.
+
+## How to run this bot?
+
+In commad line you run this and bot is working:</br>
+```python3 run_bot.py```
+
+## Add bot to your server in discord
+
+To add this bot, you just need to click this [link](https://discordapp.com/oauth2/authorize?&client_id=643102079392677898&scope=bot&permissions=34816) and choose a server. After this you are able to use your bot freely.
+
+## What can do this bot?
+
+***!generate — to generate a person</br>***
+***!ping — checks if the bot is alive. Expected answer — "pong"</br>***
+***!help — standart command which shows you all the abilities of the bot***</br>
+
+If you want to change the preffix, you can simply correct it and pass it as a parametr in **get_bot** function.
```

Comment:
This could be probably moved to a separate section, because this is ore like info for developers